webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/Info/UserFormInfoPart.ts

```diff
@@ -74,6 +74,11 @@ export class UserFormInfoPart extends FormPart<IUserFormInfoState, IUserFormStat
     return this.loaded;
   }
 
+  async disableUser() {
+    this.state.enabled = false;
+    await this.saveChanges();
```

Comment:
You need to call `save` to handle update and error handling logic (for example, if the user made some changes that will not pass validation, calling saveChanges directly will skip this logic, and it also will make it possible to run multiple concurrent save requests to the backend